test/language/expressions/arrow-function/arrow/apply_0.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: invoke arrow function using apply
+es6id: 14.2
+---*/
+
+var a = (x) => x;
+
+assert.sameValue(typeof a.apply, "function", "Apply is not a member exression on arrow function (x) => x.");

```

Comment:
Also: 

- The test description says "invoke arrow function using apply", when there is no invocation
- The assert message says "Apply is not a member exression on arrow function (x) => x.", but the assertion itself is testing that an arrow function object _does_ have a property named `apply` that _is_ a function.


I agree with @ianwjhalliday, these should be replaced by a test that asserts 

```js
Object.getPrototypeOf(a) === Function.prototype
```